### ISSUE TITLE
Admin side : Address - street field  

### ISSUE THREAD

**Issue by kokila2000** [magento/magento2#37638]
### Preconditions and environment

Admin Side - 
here i want to give a validation for all the lines in the street field and i want to give lable to street lines
like 
street[0] - Houses Number
street[1]- street address
like this i want to give in admin side

![image](https://github.com/magento/magento2/assets/77538586/608fc3bf-4d09-462a-8dcc-6234b1a5fb56)
\

https://github.com/magento/magento2/assets/77538586/434b1c98-b990-4917-9a52-794225241c69

### Steps to reproduce

Login Admin  -> customer -> all customer ->  edit the any one customer -> in customer information go to address -> here you add new add address or edit address 
in that ui compenent  street field  i want to give a validation for all the lines in the street field and i want to give lable to street lines
like 
street[0] - Houses Number
street[1]- street address
like this i want to give in admin side

### Expected result

Houses Number : 
Street Address :
and both lines are mandatory

### Actual result

street address:
only first line is mandatory 

### Triage and priority

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users without workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by devchris79** [comment]
Not every address has a number and there are also other cases like post office box addresses.
Rigid formatting needs to be carefully considered as it can cause issues.

I think the admin form is fine, it makes the user aware that the street address is required, but allows for a single line or the second line can be populated for longer addresses.

**Comment by engcom-Bravo** [comment]
@magento give me 2.4-develop instance

**Comment by kokilanavabrind** [comment]
> Not every address has a number and there are also other cases like post office box addresses. Rigid formatting needs to be carefully considered as it can cause issues.
> 
> I think the admin form is fine, it makes the user aware that the street address is required, but allows for a single line or the second line can be populated for longer addresses.

yes your correct but i have completed same thing for frontend but now i want to do same in admin also for all place in admin where we show address street field  but i trying i could able to find the were the field it coming

**Comment by chernenm** [comment]
@magento give me 2.4-dev instance

**Comment by engcom-Bravo** [comment]
Hi @kokila2000,

Thank you for reporting and collaboration.

As per Magento [devdocs](https://experienceleague.adobe.com/docs/commerce-admin/catalog/categories/create/category-create.html?lang=en) there is no option for making Number of Lines in a Street Address as Required. We are considering this as Feature request as it is an existing behavior of Magento.


Thanks.

**Comment by kokila2000** [comment]
Magento ver. 2.4.5-p1
